flask_static_digest/cli.py

```diff
@@ -1,21 +1,54 @@
-import click
+import sys
 
+import click
 from flask import current_app
 from flask.cli import with_appcontext
 
-from flask_static_digest.digester import compile as _compile
 from flask_static_digest.digester import clean as _clean
+from flask_static_digest.digester import compile as _compile
 
 
 @click.group()
-def digest():
-    """md5 tag and gzip static files."""
-    pass
+@click.pass_context
+@with_appcontext
+def digest(ctx):
+    """md5 tag and compress static files."""
+
+    ctx.ensure_object(dict)
+
+    ctx.obj["gzip"] = False
+    ctx.obj["brotli"] = False
+    ctx.obj["blacklist_filter"] = current_app.config.get(
+        "FLASK_STATIC_DIGEST_BLACKLIST_FILTER"
+    )
+
+    compression = current_app.config.get("FLASK_STATIC_DIGEST_COMPRESSION")
```

Comment:
With the other comments addressed I think we need to go back to converting this to a set and then back to a list?

Such as:

```py
    compression = list(set(current_app.config.get("FLASK_STATIC_DIGEST_COMPRESSION")))
```